### ISSUE TITLE
Limit to one repo?

### ISSUE THREAD

**Issue by pabzm** [Sojusan/github-action-reminder#4]
Thank you for this bot, I really like it!

Unfortunately it not only posts on the repo it is configured in but also on forks. I would like to avoid this since I don't want to push our development tools down other people's throat.

Is there any way to disable that behaviour?

**Comment by Sojusan** [comment]
Thank you for using my GitHub Action!

As I understand the issue:
1. The job is configured in the main repo.
1. It is working fine, just checking the PR from the main repo.
1. Someone is creating the fork of our main repo and all GitHub Workflows are also copied.
1. My action is running on the forked repository, which results in additional notifications for people configured as `default_users_to_notify` and the additional comments in the PR in the forks.

I have investigated such a scenario and there is a solution for that.
I have updated the `README.md` with a new configuration of my action to prevent such a scenario according to the [GitHub documentation - only run job for specific repository](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-only-run-job-for-specific-repository).
